.eslintrc

```diff
@@ -74,6 +75,20 @@
     "object-curly-spacing": [2, "always"],
     "quotes": [2, "single", "avoid-escape"],
     "radix": 2,
+    "react/display-name": 0,
+    "react/jsx-quotes": [2, "double", "avoid-escape"],
```

Comment:
I'm inclined to leave this as double because the JSX is like HTML markup rather than Javascript. Any JS written within the JSX file still has to follow the single quote rule it's only the actual JSX markup that doesn't. This follows the double quote standard that the React documentation appears to follow (single for JS, double for markup). What are your thoughts?